src/jit/flowgraph.cpp

```diff
@@ -21486,6 +21496,92 @@ void                Compiler::fgInline()
 #endif // DEBUG
 }
 
+#ifdef DEBUG
+
+//------------------------------------------------------------------------
+// fgNotInlineCandidate: walk tree to scan for calls that were left
+// embedded because they weren't inline candidates.
```

Comment:
Typo: This is the header for fgFindNotInlineCandidate.
Minor suggestion: I think the comment should say "tree walk helper that determines if the tree is not an inline candidate" or something. Otherwise, the plural of "scan for calls" and the singular of fgNotInlineCandidate seem at odds.